.github/workflows/ci.yml

```diff
@@ -60,6 +65,12 @@ jobs:
       - name: Install LLVM
         uses: ./.github/actions/install-llvm
 
+      - name: Install packages
```

Comment:
If we already run `cargo check`, do we also need to run `cargo test` for all examples?

Reply:
Not unless we also have tests in there, which seems weird.

Reply:
I realised that we should actually:
- compile Mun examples
- run Rust examples that marshal the generated `*.munlib`s to validate that the APIs match

This can happen in a separate PR though.

Reply:
The Windows worker runs out of memory when we use `--all-targets` for `cargo test`.

I removed benchmarks. We might want to re-add this as a separate benchmark step in our release CI